README.md

```diff
@@ -14,20 +14,21 @@ This action is used to create release from commit history with
-| `open-ai-token`                     | Experimental feature see section [experimental feature](#experimental-feature).| `token`                         | NO_DEFAULT     | no       |
+| Name                                | Description                                                                     | Example                       | Default        | Required |
+|-------------------------------------|---------------------------------------------------------------------------------|-------------------------------|----------------|----------|
+| `github-token`                      | Github token with repository scope permissions                                  | `${{ secrets.GITHUB_TOKEN }}` | NO DEFAULT     | yes      |
+| `changelog-scopes`                  | Scopes, that will be show in changelog                                          | `{"Worker": ["worker"]}`      | NO DEFAULT     | yes      |
+| `changelog-method`                  | Source from which to create changelog                                           | `pull_request_description`    | NO DEFAULT     | yes      |
+| `slack-token`                       | Slack token with `chat.postMessage` permissions                                 | `${{ secrets.SLACK_TOKEN }}`  | NO DEFAULT     | no       |
+| `slack-channel`                     | Slack channel ID                                                                | `XXXXXX`                      | NO DEFAULT     | no       |
+| `release-name-method`               | Whether to fetch release name from branch name or bump minor of last release    | `branch`                      | `branch`       | no       |
+| `create-release-pull-request`       | Whether to create release pull request                                          | `true`                        | `false`        | no       |
+| `create-github-release      `       | Whether to create github release                                                | `true`                        | `false`        | no       |
+| `base-branch`                       | Based branch where pull request will be created                                 | `master`                      | `master`       | no       |
+| `release-name-prefix`               | Prepend prefix to release name (version)                                        | `v`                           | `v`            | no       |
+| `github-changelog-file-destination` | Where to store github changelog on filesystem                                   | `github_changelog.md`         | `changelog.md` | no       |
+| `open-ai-token`                     | Experimental feature see section [experimental feature](#experimental-feature). | `token`                       | NO_DEFAULT     | no       |
+| `fetch-author-slack-ids`            | Fetch Slack IDs of commit authors in the changelog                              | `true`                        | `false`        | no       |
```

Comment:
Make note, here or in code, that `slack-token` needs to be set once this is true.

Reply:
:+1: